src/simulation/dynamics/constraintEffector/constraintDynamicEffector.cpp

```diff
@@ -140,8 +175,8 @@ void ConstraintDynamicEffector::linkInStates(DynParamManager& states)
 
 /*! This method computes the forces on torques on each spacecraft body.
  @return void
- @param integTime Integration time
- @param timeStep Current integration time step used
+ @param integTime 
```

Comment:
Why are parameter descriptions removed?

Reply:
Sorry for the oversight. Have included them back now.